infra/modules/azure_github_environment_bootstrap/id_infra_ci_iam.tf

```diff
@@ -89,3 +89,11 @@ resource "azurerm_role_assignment" "infra_ci_rg_ext_pagopa_dns_reader" {
   principal_id         = azurerm_user_assigned_identity.infra_ci.principal_id
   description          = "Allow ${var.repository.name} Infra CI identity to read DNS Zone records at resource group level"
 }
+
+# API Management
+resource "azurerm_role_assignment" "infra_ci_subscription_apim_secrets" {
```

Comment:
Thanks for adding! However, we already have an optional reference to the subscription APIM as variable, you can use the calculated count property like it has been done here:
https://github.com/pagopa/dx/blob/ac3c90b61c0870a62cb4f5c04e366716d719f56e/infra/modules/azure_github_environment_bootstrap/id_infra_cd_iam.tf#L48